pages/api/list.js

```diff
@@ -0,0 +1,25 @@
+import Field from "../../models/fellow"
+import auth0 from "../../utils/auth"
+import dbConnect from "../../utils/db"
+
+export default async function list(req, res) {
```

Comment:
Is it possible to wrap the contents of this function in a try-catch block? This would prevent any uncaught promise errors.